_posts/2018-05-07-A-Machine-Learning-Example-For-Business.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: A Machine Learning Example For Business
+author: mikaelahonen
+excerpt: A practical example of predictive machine learning. Targeted for business people how want to harness machine learning for business but also understand how it actually works.
+tags:
+- machine learning
+- business
+- predictive analytics
+- data science
+---
+
+## Introduction
+A practical example of predictive machine learning.
+Targeted for business people how want to harness machine learning for business but also understand how it actually works.
+
+The aim is to demonstrate:
+* A concrete example without going too deep in theory.
+* The difference between machine learning and reporting.
+* Benefits to business.
+
+For those who are interested in technical part, the `R` code and data is available at the end of the post.
+
+<blockquote>
+According to a definition
+<b>machine learning means
+that a machine can learn without having explicitly
+programmed for the task</b>.
+This doesn't mean that the program code would evolve by itself, but
+different results are produces in different situations depending on the data
+and the set of rules that are applied.
+</blockquote>
+
+## Machine learning use cases
+Let's quickly see some machine learning use cases.
+
+### #1 Predicting by variables
+For example predicting number of visitors in an event according to
+weather, day of week and ticket price. In the simplest case
+the algorithm might just naively count average number of visitor in the similar cases in the past.
+
+![Predicting by variables](/img/a-machine-learning-example-for-business/visitors.png)
+
+### #2 Clustering
+Customers can be segmented automatically according to
+selected variables. The algorithm might not need any input
+to make the grouping.
```

Comment:
Well, there are parameter-free clustering algorithms, but for most real-world cases, they are rarely applied. I would also include a similar example as with above (e.g. clustering a set of customers into target market categories).

Reply:
Changed the expression to "The algorithm usually needs some initial parameters such as the number of clusters to identify.". I tried to provide similar example than the first one, but I will tune the image a little bit.